src/types/on-call/index.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Date range type
+ */
+export type DateRange = {
+  start_date: string,
+  end_date: string
+};
```

Comment:
This is not used in the oncall lambdas, can you leave this in the types/index.ts and I'll get the trainees to create the appropriate folder structure.